src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts

```diff
@@ -41,6 +44,7 @@ export class DotPersonaSelectorComponent implements OnInit, OnChanges {
     @ViewChild('searchableDropdown')
     searchableDropdown: SearchableDropdownComponent;
 
+    currentOffset = 0;
```

Comment:
is this what solves `Bryan's issue` ?

Reply:
Removing, what a first approach I was trying.